src/main/java/Deadline.java

```diff
@@ -0,0 +1,17 @@
+public class Deadline extends Task {
+
+    public String deadline;
+    public Deadline(String name, String deadline) {
+        super(name);
+        this.deadline = deadline;
+    }
+    @Override
+    public String toString() {
+        String checkbox = "[ ]";
+        String typeIndicator = null;
+        if(status){
+            checkbox = "[X]";
+        }
+        return "[D]" + checkbox + " " + name + " (by:" + deadline + ")";
```

Comment:
Perhaps the checkbox variable can be stored in the parent class instead as it is not unique to the child class.(Deadline, ToDo and Event share the attribute)